### ISSUE TITLE
FieldBitmap is not accessible

### ISSUE THREAD

**Issue by mikeharv** [google/blockly-samples#2752]
### Check for duplicates

- [ ] I have searched for similar issues before opening a new one.

### Component

field-bitmap

### Description

`FieldBitmap` is not keyboard navigable or screen reader compatible. `FieldGridDropdown` does have this capability so it should be possible to use the grid classes from it. Note that the former class cannot extend the latter as it is not meant to behave like a dropdown field with a single selectable value.

### Reproduction steps

1.
2.
3.

### Stack trace

```shell

```
